gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -89,10 +92,27 @@ def add_arguments(self, parser):
               'variants. This is useful when there are header fields in '
               'variants not defined in the header sections.'))
 
+  def validate(self, parsed_args):
+    input_pattern_re_match = re.match(
+        self._GCS_URI, parsed_args.input_pattern)
+    if not input_pattern_re_match:
+      raise ValueError(
+          'Expected a file input_pattern (SCHEME://BUCKET/OBJECT) '
+          'instead of {}.'.format(parsed_args.input_pattern))
+
+    # Gets at most one pattern match result of type `filesystems.MatchResult`
+    first_match = FileSystems.match([parsed_args.input_pattern], [1])[0]
+    if not first_match.metadata_list:
+      raise ValueError(
+          'File pattern {} returned '
+          'no matched files.'.format(parsed_args.input_pattern))
```

Comment:
Could you please also catch the exception where you don't have access to the source? you can test with `--input_pattern=gs://test/*.vcf`.

Reply:
Done.